src/components/VTooltip/VTooltip.js

```diff
@@ -5,11 +5,12 @@ import Colorable from '../../mixins/colorable'
 import Detachable from '../../mixins/detachable'
 import Menuable from '../../mixins/menuable'
 import Toggleable from '../../mixins/toggleable'
+import Dependent from '../../mixins/dependent'
 
 export default {
   name: 'v-tooltip',
 
-  mixins: [Colorable, Detachable, Menuable, Toggleable],
+  mixins: [Colorable, Detachable, Menuable, Toggleable, Dependent],
```

Comment:
Update to match this spec https://docs.google.com/document/d/1xWWnAw2zYce3sVvdiRC0OuIhNx1PCt-oCNzTXeREFaw/edit?usp=sharing